docs/root/configuration/overview/v2_overview.rst

```diff
@@ -421,9 +421,70 @@ for the service definition. This is used by Envoy as a client when
             cluster_name: some_xds_cluster
 
 is set in the :ref:`rds
-<envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.rds>` field of the :ref:`HttpConnectionManager
+<envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.rds>` field
+of the :ref:`HttpConnectionManager
+<envoy_api_msg_config.filter.network.http_connection_manager.v2.HttpConnectionManager>` config.
+
+.. http:post:: /envoy.api.v2.ScopedRoutesDiscoveryService/StreamScopedRoutes
```

Comment:
Do we want to switch to using protodoc for service documentation? These seem more like pseudo-REST than gRPC..

Reply:
This is tracked in https://github.com/envoyproxy/envoy/issues/3091. I agree this should all be auto generated, but I was just trying to get us to parity for now.